www/vrbj04/cv03/Registration.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+final class Registration
+{
+    public const GENDER_MALE = true;
+    public const GENDER_FEMALE = false;
```

Comment:
ja bych  sem daval 'M' a 'F', true a false je moc diskriminujici

Reply:
Ono to asi v kontextu předávání toho parametrů dává větší smysl 👍

Reply:
https://github.com/nvbach91/4IZ278-2020-2021-LS/pull/87